Example/Simplify.swift

```diff
@@ -0,0 +1,201 @@
+//
+//  Simplify.swift
+//
+//  Copyright (c) 2018 Tomislav Filipcic <tf@7sols.com>
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files (the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions:
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+
+import Foundation
```

Comment:
Personally, I'm not really fond of storing 3rdparty sources as is. Not sure it's good idea to have separate dependency either. It'd be great if we can have our own implementation of simplification functionality.

Reply:
Agreed. It isn't something I am very familiar with though. @1ec5 - Is there a mapbox implementation that we could port over to the Nav SDK so I could drop any form of dependency?

Reply:
Will move to a simplify algorithm I added to Turf: https://github.com/mapbox/turf-swift/pull/124
Once that PR is approved and merged I will update the Turf version in Nav SDK and use it instead of another dependency.

Reply:
Turf v1.2.0-beta.1 has been released. MapboxDirections doesn’t require it yet, but you can have the navigation SDK require it by modifying the Cartfile in this repository. The CocoaPods integration test will fail until there’s a final release of Turf v1.2.0, but at least you’ll be able to prepare this PR for the moment when that happens.

```diff
diff --git a/Cartfile b/Cartfile
index ba3e3e59..1cf9c2ca 100644
--- a/Cartfile
+++ b/Cartfile
@@ -3,7 +3,7 @@ binary "https://api.mapbox.com/downloads/v2/carthage/mobile-navigation-native/Ma
 binary "https://api.mapbox.com/downloads/v2/carthage/mobile-maps/mapbox-ios-sdk-dynamic.json" ~> 6.0
 binary "https://api.mapbox.com/downloads/v2/carthage/mapbox-common/MapboxCommon-ios.json" == 7.1.2
 github "mapbox/mapbox-directions-swift" ~> 1.0
-github "mapbox/turf-swift" ~> 1.0
+github "mapbox/turf-swift" ~> 1.2.0-beta.1
 github "mapbox/mapbox-events-ios" ~> 0.10.2
 github "ceeK/Solar" ~> 2.1.0
 github "mapbox/mapbox-speech-swift" ~> 1.0
diff --git a/Cartfile.resolved b/Cartfile.resolved
index 1b02b2c1..6042bedd 100644
--- a/Cartfile.resolved
+++ b/Cartfile.resolved
@@ -10,8 +10,8 @@ github "ceeK/Solar" "2.1.0"
 github "linksmt/OHHTTPStubs" "563f48d3fab84ef04639649c770b00f4fa502cca"
 github "mapbox/MapboxGeocoder.swift" "v0.10.2"
 github "mapbox/mapbox-directions-swift" "v1.1.0"
-github "mapbox/mapbox-events-ios" "v0.10.6"
+github "mapbox/mapbox-events-ios" "v0.10.7"
 github "mapbox/mapbox-speech-swift" "v1.0.0"
-github "mapbox/turf-swift" "v1.1.0"
+github "mapbox/turf-swift" "v1.2.0-beta.1"
 github "raphaelmor/Polyline" "v5.0.2"
 github "uber/ios-snapshot-test-case" "5.0.2"
```